Let zstdgrep use GNU grep. This is Solaris specific patch, not suitable
for upstream.

--- zstd-1.5.7/programs/zstdgrep
+++ zstd-1.5.7/programs/zstdgrep
@@ -22,7 +22,7 @@ # THEORY OF LIABILITY, WHETHER IN CONTRA
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-grep=${GREP:-grep}
+grep=${GREP:-ggrep}
 zcat=${ZCAT:-zstdcat}
 
 endofopts=0
